_posts/2025-10-02-why-we-choose-what-we-choose.md

```diff
@@ -0,0 +1,62 @@
+---
+layout: single
+title: "Why We Choose What We Choose"
+excerpt: "In selecting one way to package Python, pyOpenSci chooses between a lot of code tools. These choices often come after months of exploration and debate. Find out what motivates us to make the decisions that we do."
+author: "Jeremiah Paige"
+permalink: /blog/pyopensci-why-we-choose.html
+categories:
+  - blog-post
+  - community
+classes: wide
+toc: true
+comments: true
+last_modified: 2025-10-02
+---
+
+A primary focus at pyOpenSci, one of our petals of support, is selecting packaging tools that work well for our users and work well together. We use our curated selection of tooling in our packaging guide, in our online tutorials, and in our trainings. We don’t require the use of any of our selected tools to submit a package to our peer review program but we will suggest them if the package authors ask for help trying to clean up or adopt new workflows.
+
+The members of pyOpenSci spend a great deal of time selecting these tools, debating tradeoffs, test-driving them in new situations, looking at new community trends, and listening to feedback from events we run. But up until now we haven't typically posted in the open either how or why we have come to the decisions we did. This post outlines, in not very strict terms, the rubric we use when selecting a project that we want to recommend. We focus on a beginner-leaning happy path for packaging workflows. In other words, what works really well for new projects that don't have a lot, if any, non-conventional requirements to share their code.
+
+Our rubric comes from five categories, approximately ordered as follows:
+
+## Tools That Are Free and Open
+
+We love open software! It's kind of in our name. We are always looking to nurture and support open source software, even beyond packaging projects. It should come as no surprise then, that we only choose tools in our packaging guide that are themselves open source.
+
+We don’t just appreciate open source though, we also look for projects that are open contribution – that do most of their maintenance, stewardship, and designing in public. This means that there is a public bug tracker, that new issues to that bug tracker are accepted from anyone, and also fixes for those bugs are accepted from non-maintainers. It may also mean that new features ideas are accepted from the community, or even given a period of public comment.
```

Comment:
I think "new feature ideas" is more grammatically correct than "new feature**s** ideas." Alternatively, "ideas for new features" if you want to be plural for both idea and feature.